src/css/notmalise.css

```diff
@@ -0,0 +1,5 @@
+.product-card {
```

Comment:
`normalise.css` should be used to set some standard styles to tags all over the project. 
Here you style just one block, so name the file by that block.